src/test/unit/test.js

```diff
@@ -491,6 +491,51 @@ describe('Client', function() {
     })
   })
```

Comment:
Yes, definitely, this makes more sense than what I had in mind :) Thanks for the explanation, I'll fix the tests up tomorrow morning.